LethalAPI.Core/Patches/GameNetworkManagerJoinPatch.cs

```diff
@@ -33,42 +35,50 @@ private static void SteamMatchmaking_OnLobbyCreated_Postfix(Result result, ref L
         lobby.SetData("__joinable", lobby.GetData("joinable")); // the actual joinable
 
         // if the user is forced to only allow modded user to join, joinable flag is set to prevent vanilla user to join
-        if (Core._moddedOnly)
+        if (ModdedLobbyManager.ModdedOnly)
         {
             lobby.SetData("joinable", "false");
         }
     }
+}
 
-    [HarmonyPatch(nameof(GameNetworkManager.LobbyDataIsJoinable))]
+/// <summary>
+/// Patches the <see cref="GameNetworkManager.LobbyDataIsJoinable"/> to ensure modded players only join.
```

Comment:
This should be changed to "Patches the .... to replace the game lobby checking to ensure modded player only join for a modded lobby when they are forced to"